### PR TITLE
adding some more content to ember/v1

### PR DESCRIPTION
**Pull request by serenaf**
This is sweaty work 😅  These are most of the deprecations for ember/v1

### CODE REVIEW

File: content/ember/v1/access-to-instances.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+---
+id: deprecate-access-to-instances-in-initializers
+title:  Access to Instances in Initializers
+until: ''
```
Comment (mansona) [inline content/ember/v1/access-to-instances.md]: what's the story with all these empty "until" ? 

if they don't have an until can't we just delete the line so it comes through as null?

File: content/ember/v1/access-to-instances.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+Previously, initializers had access to an object that allowed them to
+both register new classes and get instances of those classes.
+
+If you have an initializer that gets instances of a class, you need to
+change it to use an instance initializer.
+
+Change code that looks like this:
+
+```js
```
Comment (mansona) [inline content/ember/v1/access-to-instances.md]: Needs to be updated to new syntax for code highlighting, if you would like I can contribute my regexing 💪 😂

File: content/ember/v1/access-to-instances.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    var clock = instance.container.lookup("clock:main");
+    clock.setStartTime(Date.now());
+  }
+});
+```
+
+Added in [PR #10256](https://github.com/emberjs/ember.js/pull/10256).
+
+For more information, see [the instance initializer deprecation
+guide](../instance-initializers).
```
Comment (mansona) [inline content/ember/v1/access-to-instances.md]: eh... relative links? 😳 

😭 

We should have a talk about these, maybe convert them to #links or something

File: content/ember/v1/ember-collection-view.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+Legacy support of `Ember.CollectionView` will be provided via the [ember-legacy-views](https://github.com/emberjs/ember-legacy-views) addon.
+
+##### Migrating away from `Ember.CollectionView`
+
+In most cases collection view can be replaced using the `{{each}}` helper in combination with the `{{component}}` helper.
+
+To be able to programmatically decide which component to use for an item, you can use a `Ember.Helper`
+
+```app/helpers/animal-component.js
```
Comment (mansona) [inline content/ember/v1/ember-collection-view.md]: needs new data-filename implementation

File: content/ember/v1/access-to-instances.md (hunk 1 above)
Comment (serenaf) [reply content/ember/v1/access-to-instances.md]: good point, the problem is that it seems ember deprecations v1 do not have an until field, see https://www.emberjs.com/deprecations/v1.x/#toc_observing-container-views-like-arrays vs 
https://www.emberjs.com/deprecations/v2.x/#toc_ember-string-fmt 😩

File: content/ember/v1/access-to-instances.md (hunk 2 above)
Comment (serenaf) [reply content/ember/v1/access-to-instances.md]: that would be amazing ;) I keep forgetting we need to add this new syntax :) definitly one to add to the list of readme for new contributors :)

File: content/ember/v1/access-to-instances.md (hunk 3 above)
Comment (serenaf) [reply content/ember/v1/access-to-instances.md]: 💯 What would you recommend? I thing converting them into #links with link to the relevant id

Comment (mansona) [reply content/ember/v1/access-to-instances.md]: https://github.com/ember-learn/deprecation-app/issues/41

**Comment by serenaf** [comment]
cc @mansona rebased from my (broken 🙈  master

**Pull request merged**